README.md

```diff
@@ -77,30 +77,30 @@ latest crate code is, the crate can be in one of 3 states:
 
 These are the empty repositories that have PRs waiting to be merged.
 
-- [linux-loader](https://github.com/rust-vmm/linux-loader): parser and loader
-  for vmlinux and bzImage images as well as some other helpers for kernel
-  commandline.
-- [virtio-bindings](https://github.com/rust-vmm/virtio-bindings): Rust FFI
-  bindings to virtio kernel headers generated using
-  [bindgen](https://crates.io/crates/bindgen).
 - [vmm-vcpu](https://github.com/rust-vmm/vmm-vcpu/): a hypervisor-agnostic
   abstraction for Virtual CPUs (vCPUs).
 
 ### rust-vmm
 
-- [vm-memory](https://github.com/rust-vmm/vm-memory): abstractions over a
-  virtual machine's memory.
 - [vm-virtio](https://github.com/rust-vmm/vm-virtio/): virtio device trait and
   implementation for virtio primitives such as virtqueues and descriptor chain.
-- [vmm-sys-util](https://github.com/rust-vmm/vmm-sys-util/): collection of
-  modules providing helpers and utilities for building VMMs and hypervisors.
 
 #### crates.io
 
 - [kvm-bindings](https://crates.io/crates/kvm-bindings): Rust FFI bindings
   to KVM generated using [bindgen](https://crates.io/crates/bindgen).
 - [kvm-ioctls](https://crates.io/crates/kvm-ioctls): Safe wrappers over the
   KVM API.
+- [linux-loader](https://github.com/rust-vmm/linux-loader): parser and loader
+  for vmlinux and bzImage images as well as some other helpers for kernel
+  commandline.
+- [virtio-bindings](https://github.com/rust-vmm/virtio-bindings): Rust FFI
+  bindings to virtio kernel headers generated using
+  [bindgen](https://crates.io/crates/bindgen).
+- [vm-memory](https://github.com/rust-vmm/vm-memory): abstractions over a
+  virtual machine's memory.
+- [vmm-sys-util](https://github.com/rust-vmm/vmm-sys-util/): collection of
+  modules providing helpers and utilities for building VMMs and hypervisors.
```

Comment:
There's one more crate, vfio-bindings. Could you please also help to include it?